src/game/overlay.c

```diff
@@ -78,12 +80,90 @@ static RGB888 m_ColorBarMap[][COLOR_STEPS] = {
       { 120, 60, 70 } },
 };
 
+typedef struct BAR_INFO {
+    int32_t type;
+    int32_t value;
+    int32_t max_value;
+    int32_t show;
+    int32_t show_mode;
```

Comment:
Consider making this a `T1M_BAR_SHOW_MODE`

Reply:
Didn't even realize / think to do that lol. Mind blown.